.circleci/config.yml

```diff
@@ -242,9 +242,12 @@ jobs:
           name: install pip
           command: |
             apt-get update -q
-            apt-get install -q -y python-pip
+            apt-get install -q -y python-pip python3-pip
       - run: pip install flake8==3.4.1
```

Comment:
We should probably upgrade this version too.. (otherwise we could end up with conflicting warnings).    Maybe either upstate it as part of this CL or open a separate one if you prefer?

Reply:
Seems like #8877 was created for this.

Reply:
8877 only ran flake8 on legacy Python.